src/components/molecules/InternalChevronAccordion/InternalChevronAccordion.tsx

```diff
@@ -100,9 +100,10 @@ const Accordion = ({
             transform: isOpen ? "rotate(180deg)" : "none",
             transition: "all 0.3s ease 0s",
           }}
+          $mr={"space-between-xs"}
         />
       </StyledAccordionButton>
-      <InternalAccordionContent aria-labelledby={id}>
+      <InternalAccordionContent aria-labelledby={id} $overflow={"scroll"}>
```

Comment:
Again, this only affects `OakOutlineAccordion` and only has effect if we set the `height` of accordion. The default was `overflow: visible` but I imagine with any overflow on accordion we want it to be `scroll` anyways